pipeline: outputs: cloudwatch: Add StorageResolution Support

devin-andrenam opened the pull request:
Documentation update in association with according PR to fluent bit.

devin-andrenam committed:
pipeline: outputs: cloudwatch: Add StorageResolution Support

Signed-off-by: Devin Stafford <devin@andrenam.com>

esmerel reviewed (approved):
Minor stylistic change. A @fluent/fluent-bit-maintainers should doublecheck the definition for technical accuracy, but I'm otherwise ok with this.

esmerel committed:
Update pipeline/outputs/cloudwatch.md

Signed-off-by: Lynette  Miles <6818907+esmerel@users.noreply.github.com>

eschabell commented:
@devin-andrenam while waiting on the code PR changes https://github.com/fluent/fluent-bit/pull/10398 you need to make, would you please fix the conflicts here so I'm ready to merge your docs update when we can?

esmerel committed:
Merge branch 'master' into devin/update-docs-storage-resolution-emf

Signed-off-by: Lynette  Miles <6818907+esmerel@users.noreply.github.com>

patrick-stephens commented:
There are on changes?

esmerel committed:
Update cloudwatch.md

Signed-off-by: Lynette  Miles <6818907+esmerel@users.noreply.github.com>

esmerel commented:
Appears I overwrote the change fixing the conflict, I corrected it. I was afraid I'd done that at least once.

eschabell reviewed (changes requested):
This has too many backticks:   

```
 `metric`_storage_resolution` 
```

eschabell committed:
pipeline: outputs: cloudwatch: general doc updates and cleanup

- Merge master into PR branch
- Fix malformed metric_storage_resolution row (backticks, missing
  default column, default value, "1s/60s" -> "1/60")

Signed-off-by: Eric D. Schabell <eric@schabell.org>

eschabell reviewed (approved):
@devin-andrenam waiting for code PR merging.
